demo/gen.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+mkdir -p src/test/java/foo
+for i in {00..99}; do
+cat > src/test/java/foo/Hello${i}Test.java << EOF
+package foo;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class Hello${i}Test {
+    @Test public void one() {
+        if (Math.random() < 0.015) {
+            fail("oops");
+        }
+    }
+    @Test public void two() {}
+    @Test public void three() throws Exception {
+        Thread.sleep(${i##0}0);
+    }
+    @Test public void four() throws Exception {
+        Thread.sleep(1000);
+    }
+}
+EOF
+done
```

Comment:
Yeah the `${i##0}0` trick took me a while…copied from `parallel-test-executor-plugin-samples` repo BTW (now only used by `WorkflowPluginTest` in ATH).